ms-engine/src/main/java/net/trajano/ms/engine/SpringJaxRsHandler.java

```diff
@@ -86,7 +89,9 @@
      * @param applicationClasses
      *            application classes
      * @return the handlers
+     * @deprecated use {@link JaxRsRouter}
      */
+    @Deprecated
     @SafeVarargs
     public static SpringJaxRsHandler[] multipleRegisterToRouter(final Router router,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)